README.md

```diff
@@ -86,17 +86,10 @@ Avoiding code duplication is **not** a goal. **Readability** and **hackability**
 
 ## Setup
 
-Clone the repo:
-
-```bash
-git clone https://github.com/Lightning-AI/litgpt
-cd litgpt
-```
-
 Install with all dependencies (including CLI, quantization, tokenizers for all models, etc.):
 
 ```bash
-pip install -r requirements-all.txt
+pip install 'litgpt[all]'
```

Comment:
Oh, so if all is always needed, then maybe no need for this in the first place? Or is the plan to relax it later?

Reply:
All is not always needed, but since we don't know what exactly the user will run, we recommend installing all in the README.